app/_posts/2023-02-01-kuma-2-1-0.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Kuma 2.1 released with full suite of next-gen policies
+description: Next gen policy, cross control-plane API and UI improvements. 
+date: 2023-02-01
+headerImage: /assets/images/blog/kuma_2_1_blog_banner.jpg
+tags:
+  - Release
+---
+
+We are excited to announce the release of Kuma 2.1! In this release, we’re shipping the full suite of new and improved policies announced (and started) in 2.0.
+Additionally, we’re launching some more great UX improvements in the UI and a host of smaller fixes. 
+
+In order to take advantage of the latest and greatest in service mesh, we strongly suggest [upgrading to Kuma 2.1](https://github.com/kumahq/kuma/releases/tag/2.1.0). Upgrading is easy through `kumactl` or Helm and doesn’t require any special steps.
```

Comment:
```suggestion
In order to take advantage of the latest and greatest features in service mesh, we strongly suggest [upgrading to Kuma 2.1](https://github.com/kumahq/kuma/releases/tag/2.1.0). Upgrading is easy through `kumactl` or Helm and doesn’t require any special steps.
```

Reply:
this is our standard footer going to keep it as is